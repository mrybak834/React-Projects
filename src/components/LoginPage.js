import React from 'react';
import { connect } from 'react-redux';

// Stateless functional component
const LoginPage = () => (
    <div>
        <button>Login</button>
    </div>
);

export default LoginPage;

// Class based component with redux
// export class LoginPage extends React.Component {

//     render() {
//         return (
//             <div>
//                 <button>Login</button>
//             </div>
//         );
//     }
// }


// const mapStateToProps = (state, props) => {
//     return {

//     };
// }

// export default connect(mapStateToProps)(LoginPage);