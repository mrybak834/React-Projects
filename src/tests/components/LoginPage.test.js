import React from 'react';
import { shallow } from 'enzyme';
import LoginPage from '../../components/LoginPage';

test('Component Snapshot', () => {
    const wrapper = shallow(<LoginPage />);

    expect(wrapper).toMatchSnapshot();
})