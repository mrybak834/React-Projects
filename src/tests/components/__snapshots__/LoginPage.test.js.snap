// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Snapshot 1`] = `
<div>
  <button>
    Login
  </button>
</div>
`;
